Attempting to add listener when controller is null

At first startup of the snapshot 1641, I got this error:
```

13:30:10.178 [ERROR] [.zwave.handler.ZWaveControllerHandler] - Attempting to add listener when controller is null
13:30:10.256 [INFO ] [ome.event.ThingStatusInfoChangedEvent] - 'zwave:serial_zstick:controller' changed from UNINITIALIZED to INITIALIZING
13:30:10.339 [INFO ] [smarthome.event.ThingUpdatedEvent    ] - Thing 'zwave:serial_zstick:controller' has been updated.
13:30:10.350 [INFO ] [ome.event.ThingStatusInfoChangedEvent] - 'zwave:serial_zstick:controller' changed from INITIALIZING to OFFLINE (BRIDGE_OFFLINE): Controller is offline
13:30:10.356 [INFO ] [ding.zwave.handler.ZWaveSerialHandler] - Connecting to serial port '/dev/ttyACM0'
13:30:10.382 [INFO ] [ding.zwave.handler.ZWaveSerialHandler] - Serial port is initialized
13:30:10.495 [INFO ] [ave.internal.protocol.ZWaveController] - Starting ZWave controller
13:30:10.499 [INFO ] [ave.internal.protocol.ZWaveController] - ZWave timeout is set to 5000ms. Soft reset is false.
```
By the way, the binding is working.

--- a/src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java
+++ b/src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java
@@ -503,7 +503,7 @@ public abstract class ZWaveControllerHandler extends BaseBridgeHandler implement
         }
 
         if (controller == null) {
-            logger.info("Attempting to add listener when controller is null");
+            logger.debug("Attempting to add listener when controller is null");
             return false;
         }
         controller.addEventListener(listener);

